assets/src/edit-story/app/story/storyProvider.js

```diff
@@ -36,6 +37,7 @@ function StoryProvider( { storyId, children } ) {
 	// Don't update 1 of these in an effect based off another base variable.
 	// Only update these directly as a response to user or api interactions.
 	const [ pages, setPages ] = useState( [] );
+	const [ fonts, setFonts ] = useState( [] );
```

Comment:
I don't think this belongs in the story provider. I think a new `FontProvider` would be a better choice for his logic. It would need to be a parent of the story provider in the layering in `<App>`.

Yes, the story provider needs to know which fonts (_font presets_) are used in the story, but it doesn't need to concern itself with which fonts are available nor how and when these fonts are loaded.

Reply:
Font provider now added.